sms_plusserver.py

```diff
@@ -71,7 +71,7 @@ def __init__(self, response_text):
     def __repr__(self):
         return '<{} [{}]>'.format(self.__class__.__name__, self.message or '')
 
-    def __getitem__(self, item):
+    def __getitem__(self, item: Any):
```

Comment:
I'd suggest to avoid using `Any`

Reply:
@escer any suggestion here?

Reply:
I will remove it - then

Reply:
Done with https://github.com/W-Z-FinTech-GmbH/sms_plusserver/pull/6/commits/548f9fde0d8759ba95fcb55ba2c417194e39dd2f